File: Munchkin.Shared/Models/Game.cs
namespace Munchkin.Shared.Models
{
    public class Game
    {
        public Guid Id { get; set; }
        public int TurnIndex { get; set; }
        public string? State { get; set; }
        public GameStatus Status { get; set; }
        public GameLobby Lobby { get; set; } = new();
        public Table Table { get; set; }

        public Game(Table table)
        {
            Table = table;
        }

        public bool IsPlayerTurn(Guid playerId)
            => Table.Places[TurnIndex % Table.Places.Count].Player.Id == playerId;
    }
}

File: Munchkin-UI/src/app/models/game.ts
import { GameLobby } from './game-lobby';
import { GameStatus } from './game-status';
import { GameTable as GameTable } from './game-table';

export interface Game {
  id: string;
  turnIndex: number;
  state: string;
  status: GameStatus;
  lobby: GameLobby;
  table: GameTable;
}
